src/common/translate/lang/en.json

```diff
@@ -30,6 +30,7 @@
     "MENU_GENERATE_ACCOUNT": "Generate an Account",
     "SUBMIT": "Submit",
     "NEXT": "Next",
+    "VERIFY_ACCOUNT": "Verify Account",
```

Comment:
This copy is weird, maybe we should verify this? It is also has different copy on the design?

Reply:
Oh, it seems like the design now has more steps where you confirm that you know what you are doing